src/pages/AzurePreview.js

```diff
@@ -53,17 +45,7 @@ const AzurePreview = () => {
     azurePreviewStore.set(true)
   }
 
-  // Use a Sam group to determine if a user is an Azure Preview user.
-  // This is problematic when the user needs to register/accept ToS, since that's a prerequisite
-  // for checking Sam group membership. TOAZ-301 is open to change this to a B2C check instead of Sam.
-  const loadAlphaAzureMember = withErrorIgnoring(async () => {
-    setIsAlphaAzureUser(await Ajax(signal).Groups.group(getConfig().alphaAzureGroup).isMember())
-  })
-
-  // Lifecycle
-  useOnMount(() => {
-    loadAlphaAzureMember()
-  })
+  const isAlphaAzureUser = !getConfig().isProd || getUser().allowAppAccess === 'true'
```

Comment:
In non-prod let anyone log in via Azure -- this allow-list is only a requirement for prod app.terra.bio.